[2021-03-17 16:08:48,799] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: PostgresOperator_dag.create_table 2021-03-17T16:08:47.112795+00:00 [queued]>
[2021-03-17 16:08:48,822] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: PostgresOperator_dag.create_table 2021-03-17T16:08:47.112795+00:00 [queued]>
[2021-03-17 16:08:48,823] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-17 16:08:48,824] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-17 16:08:48,826] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-17 16:08:48,837] {taskinstance.py:1063} INFO - Executing <Task(PostgresOperator): create_table> on 2021-03-17T16:08:47.112795+00:00
[2021-03-17 16:08:48,843] {standard_task_runner.py:52} INFO - Started process 42 to run task
[2021-03-17 16:08:48,851] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'PostgresOperator_dag', 'create_table', '2021-03-17T16:08:47.112795+00:00', '--job-id', '15', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first.py', '--cfg-path', '/tmp/tmpx1bwhiyh', '--error-file', '/tmp/tmperlil1am']
[2021-03-17 16:08:48,852] {standard_task_runner.py:77} INFO - Job 15: Subtask create_table
[2021-03-17 16:08:48,919] {logging_mixin.py:104} INFO - Running <TaskInstance: PostgresOperator_dag.create_table 2021-03-17T16:08:47.112795+00:00 [running]> on host c065581206d5
[2021-03-17 16:08:48,990] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=PostgresOperator_dag
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2021-03-17T16:08:47.112795+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-17T16:08:47.112795+00:00
[2021-03-17 16:08:48,992] {postgres.py:69} INFO - Executing: CREATE TABLE IF NOT EXISTS user_table(
            id integer NOT NULL, email VARCHAR, active BOOLEAN
            );
[2021-03-17 16:08:49,021] {base.py:74} INFO - Using connection to: id: postgresql. Host: db, Port: 5432, Schema: airflow, Login: postgres, Password: XXXXXXXX, extra: None
[2021-03-17 16:08:49,028] {dbapi.py:180} INFO - Running statement: CREATE TABLE IF NOT EXISTS user_table(
            id integer NOT NULL, email VARCHAR, active BOOLEAN
            );, parameters: None
[2021-03-17 16:08:49,041] {dbapi.py:186} INFO - Rows affected: -1
[2021-03-17 16:08:49,060] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=PostgresOperator_dag, task_id=create_table, execution_date=20210317T160847, start_date=20210317T160848, end_date=20210317T160849
[2021-03-17 16:08:49,101] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-17 16:08:49,148] {local_task_job.py:146} INFO - Task exited with return code 0
